### PR TITLE
Add interop for Atomics

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: node_interop/lib/atomics.dart [hunk 1]
```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2020, Anatoly Pulyaevskiy. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
+/// Bindings for the Atomics JavaScript obect. Since this object only has static
```
Comment (pulyaevskiy) [inline node_interop/lib/atomics.dart]: typo: `obect` => `object`.

File: node_interop/lib/atomics.dart [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+/// methods, it's exposed as a module so that it can be used in the standard
+/// Dart style.
```
Comment (pulyaevskiy) [inline node_interop/lib/atomics.dart]: One thing I prefer to preserve, if possible, is to keep bindings closer to the actual JS API. What do you think of adding a container `Atomics` abstract class and making the below methods static inside it? I _think_ JS interop supports external static methods, but I'm not 100% sure.

File: node_interop/lib/atomics.dart [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+@JS()
+library node_interop.atomics;
```
Comment (pulyaevskiy) [inline node_interop/lib/atomics.dart]: Also, it looks like Atomics is a global JS object, so we need to add an export of atomics in `lib/node.dart` which is where all the global stuff is exported.

**Review by pulyaevskiy — COMMENTED** [review]
Thanks, this looks good, just have a few minor comments. PTAL.

**Comment by nex3** [comment]
Moved it into `js.dart` (since it's a globally-available JS name, not Node-specific) and updated the API to match the JS API.

**Review by pulyaevskiy — APPROVED** [review]
LGTM, build failure is unrelated to this change (and is rather weird, pub installs build_modules 2.11.0 when it can install 2.11.1, which is what happens locally and resolves the issue).

Merging. Thanks again!

**Pull request merged**